src/components/Product/MaxAI/index.tsx

```diff
@@ -0,0 +1,450 @@
+            <CloudinaryImage
+                src="https://res.cloudinary.com/dmukukwp6/image/upload/short_aarrr_e2143decd4.png"
+                width={428}
+                placeholder="none"
+            />
+        ),
+    },
+]
+
+const subfeaturesItemCount = 4
+const subfeatures = [
+    {
+        title: 'Product analytics',
+        description: 'Explore your product data with graphs, funnels, retention charts, and more just by asking Max',
+        icon: <IconGraph />,
+        color: 'blue',
+    },
+    {
+        title: 'Session replays',
+        description: 'Tell Max what behaviours you are looking for and he can find the matching session recordings',
+        icon: <IconRewindPlay />,
+        color: 'yellow',
+    },
+    {
+        title: 'SQL insights',
+        description: 'Tired of writing long SQL queries? Max can do that for you to help you query your data warehouse',
+        icon: <IconCode />,
+        color: 'seagreen',
+    },
+    {
+        title: 'PostHog docs',
+        description: 'Want advice on building your product? Max can access all our docs and give you tips',
+        icon: <IconBook />,
+        color: 'green',
+    },
+]
+
+const questions = [
+    {
+        question: 'Where do my users drop off?',
+        url: 'https://app.posthog.com/#panel=max%where%20do%users%dropoff%3F',
+    },
+    {
+        question: 'What are my most popular pages?',
+        url: 'https://app.posthog.com/#panel=max%most%20popular%pages%3F',
+    },
+    {
+        question: 'What is distribution of paid vs. organic traffic?',
+        url: 'https://app.posthog.com/#panel=max%paid%20vs%organic%traffic%3F',
+    },
+    {
+        question: 'Write an SQL query for me?',
+        url: 'https://app.posthog.com/#panel=max%write%20an%sql%%3F',
+    },
+    {
+        question: 'What is my ARR?',
+        url: 'https://app.posthog.com/#panel=max%what%is%my%arr%3F',
+    },
+    {
+        question: 'How many pageviews did we get today?',
+        url: 'https://app.posthog.com/#panel=max%how%many%pageviews%today%3F',
+    },
+    {
+        question: 'Show me a signup funnel',
+        url: 'https://app.posthog.com/#panel=max%show%me%signup%funnel%3F',
+    },
+]
+
+const comparisonColumnCount = 4
+const comparison = [
+    {
+        feature: 'Understands natural language',
+        companies: {
+            ChatGPT: true,
+            Anthropic: true,
+            PostHog: true,
+        },
+    },
+    {
+        feature: 'Writes SQL queries',
+        companies: {
+            ChatGPT: true,
+            Anthropic: true,
+            PostHog: true,
+        },
+    },
+    {
+        feature: 'Builds insights',
+        companies: {
+            ChatGPT: true,
+            Anthropic: true,
+            PostHog: true,
+        },
+    },
+    {
+        feature: 'Understands your data',
+        companies: {
+            ChatGPT: false,
+            Anthropic: false,
+            PostHog: true,
+        },
+    },
```

Comment:
Let's move this to right below "Writes SQL queries", to drive it home how data understanding is key here. Could be worth clarifying to "Understands your PostHog data" or "Understands your data and schema", as just "your data" is pretty vague